cmd/libs/go2idl/client-gen/args/args.go

```diff
@@ -29,6 +29,10 @@ type Args struct {
 	// the respective types.go. We still need GroupVersions in the struct because
 	// we need an order.
 	GroupVersionToInputPath map[unversioned.GroupVersion]string
+
+	// Overrides for which resources should be included in the client.
+	IncludeResourceOverrides map[unversioned.GroupVersion][]string
```

Comment:
How about `IncludedTypeOverrided`? We are use the word "Type" throughout the client-gen.